### ISSUE TITLE
[V6] call to Array.findIndex breaks IE

### ISSUE THREAD

**Issue by pulse00** [erikras/redux-form#1326]
V6 seems to use [findIndex](https://github.com/erikras/redux-form/blob/v6/src/reducer.js#L236) which needs a [polyfill](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex) in some browsers.

I'd be happy to create a documentation PR if i'm not missing something here.

**Comment by pulse00** [comment]
Here's the es6 polyfill if someone stumbles upon this until this is clarified:

```js
if (!Array.prototype.findIndex) {
  Array.prototype.findIndex = function (predicate) {
    if (this === null) {
      throw new TypeError('Array.prototype.findIndex called on null or undefined');
    }
    if (typeof predicate !== 'function') {
      throw new TypeError('predicate must be a function');
    }
    const list = Object(this);
    const length = list.length >>> 0;
    const thisArg = arguments[1];
    let value;

    for (let i = 0; i < length; i++) {
      value = list[i];
      if (predicate.call(thisArg, value, i, list)) {
        return i;
      }
    }
    return -1;
  };
}
```

**Comment by erikras** [comment]
Thanks for pointing that out, @pulse00. I didn't write that line. 😀 I'll write that another way.

**Comment by erikras** [comment]
Fixed published in [`v6.0.0-rc.4`](https://github.com/erikras/redux-form/releases/tag/v6.0.0-rc.4).
